### ISSUE TITLE
Two types of shortcuts for which I'd like to avoid the notification

### ISSUE THREAD

**Issue by fref** [halirutan/IntelliJ-Key-Promoter-X#130]
Or at least an option to avoid the notification, if it does not please everyone:

- shortcuts for which the focus must be changed manually to be active (for instance, if I need to make the run window active to use CTRL-F5, there is no point in suggesting a shortcut as I would need to use the mouse to click to focus the area before using the keyboard)
- run configuration shortcuts (I don't want a suggestion for every new run configuration that I create and use multiple times)

**Comment by stenaksel** [comment]
I totally agree that is should be possible to avoid all annoying CTRL-F5 notifications. 
(Maybe a possibility to "edit" the registered "reruns" to skip ->  just use "*" (to avoid ALL))
